### PR TITLE
Set spatial position when autoplay is set.

### PR DESCRIPTION
**Pull request by sorskoot**
When autoplay is true the spatial position is not used when the audio starts playing, and if the object playing the audio is static, it never updates.

### CODE REVIEW

**Comment by sorskoot** [comment]
I believe this would PR #14 obsolete.

File: howler-audio-source.js [hunk 1]
```diff
@@ -28,27 +28,30 @@ export class HowlerAudioSource extends Component {
...
+        this.origin = new Float32Array(3);
+        this.lastOrigin = new Float32Array(3);
+
         this.audio = new Howl({
             src: [this.src],
             loop: this.loop,
             volume: this.volume,
-            autoplay: this.autoplay,
+            autoplay: false,
+            onload:()=>{
```
Comment (Squareys) [inline howler-audio-source.js:40]: Make sure to apply prettier formatting please 🙏

File: howler-audio-source.js [hunk 2]
```diff
@@ -28,27 +28,30 @@ export class HowlerAudioSource extends Component {
...
+                if (this.spatial) {
+                    this.object.getPositionWorld(this.origin);
+                }
+                if (this.autoplay) {
+                    this.play();
```
Comment (Squareys) [inline howler-audio-source.js:45]: Maybe `this.updatePosition()` should be called immediately after play to make sure the sound isn't emitted from an incorrect position before the first update() call?

Comment (sorskoot) [reply howler-audio-source.js:45]: Yes, I was thinking about this. Maybe it's best to start autoplay on the first update after the audio is loaded. I should also convert the file to TypeScript if I'm making edits.

**Pull request merged**